staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -255,8 +255,8 @@ func (l *CostEstimator) EstimateCallCost(function, overloadId string, target *ch
 			// Worst case size is where is that a separator of "" is used, and each char is returned as a list element.
 			max := sz.Max
 			if len(args) > 1 {
-				if c := args[1].Expr().GetConstExpr(); c != nil {
-					max = uint64(c.GetInt64Value())
+				if v := args[1].Expr().AsLiteral(); v != nil {
+					max = uint64(v.Value().(int64))
```

Comment:
Is this safe

Reply:
Yes! Previous checks already ensures the Expr is indeed an int64. However, I think it's better safe than sorry. Changing here.

Reply:
Also, the old code is more unsafe because if it is not an Int64, GetInt64Value() returns 0, causing the cost to be zero.

Reply:
Updated. PTAL. Thank you.